.jshintrc

```diff
@@ -4,7 +4,8 @@
     "window",
     "-Promise",
     "RemoteStorage",
-    "remoteStorage"
+    "remoteStorage",
```

Comment:
With the new rs build and explicit importing, we can actually remove both "RemoteStorage" and "remoteStorage" from the here.